presto-docs/src/main/sphinx/prestissimo/prestissimo-features.rst

```diff
@@ -184,3 +184,184 @@ for cleanup. Only applicable when ``enable-old-task-cleanup`` is ``true``.
 Old task is defined as a PrestoTask which has not received heartbeat for at least
 ``old-task-cleanup-ms``, or is not running and has an end time more than
 ``old-task-cleanup-ms`` ago.
+
+
+Session Properties
+------------------
+
+Defines all system session properties supported by native worker to ensure
+that they are the source of truth and to differentiate them from Java based
+session properties.
+
+The following are the native session properties kept as the source of truth in prestissimo.
+
+``driver_cpu_time_slice_limit_ms``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``1000``
+
+Native Execution only. ``driver_cpu_time_slice_limit_ms`` defines the maximum CPU time,
+in milliseconds, that a driver thread is permitted to run before it must yield to other threads,
+facilitating fair CPU usage across multiple threads.
+A positive value enforces this limit, ensuring threads do not monopolize CPU resources.
+Setting this property to ``zero`` allows a thread to run indefinitely without yielding,
```

Comment:
Consider moving this sentence "Setting this property to zero allows a thread to run indefinitely without yielding, which is not recommended in a shared environment as it can lead to resource contention. " after the sentence about negative values. I mocked this up in a local build and am including a screenshot to show what the suggestion looks like - along with adding lines before the Positive and Negative value sentences to left-justify them, break up the paragraph with negative space to make it easier for the reader to identify the values they should understand. 

(My suggestion also replaces "this property" with the property name.)

<img width="838" alt="Screenshot 2024-03-27 at 10 16 24 AM" src="https://github.com/prestodb/presto/assets/7013443/19bf5d0c-0bd6-4189-bb7f-f35af8da50bb">

Reply:
updated as suggested